### DISCUSSION TITLE
Why MahApps developers opt for strange default settings?

### DISCUSSION THREAD

**Discussion by omeraziz** [MahApps/MahApps.Metro#4136] [Ideas]
On title of MetroWindow all caps is default and no matter how many spaces you add, it will delete all spaces between two words.
On buttons, either they change the text to all caps or all smalls.

If a user want to show all caps then he can write text as all caps etc. Why they change it and force other developers to find a fix of these intentionally introduced issues?

Please leave the text as is. If someone has to change the case of text he will do it himself, you don't have to worry about it.

Now, how can I add space between two words in MetroWindow Title? I want to display "Application Pro"

**Comment by punker76** [comment]
> Now, how can I add space between two words in MetroWindow Title? I want to display "Application Pro"

Just set Title and (optional, for normal case) the TitleCharacterCasing:

```xaml
<mah:MetroWindow x:Class="MetroDemo.MainWindow"
                 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
                 xmlns:mah="http://metro.mahapps.com/winfx/xaml/controls"
                 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"

                 Title="Application Pro"
                 TitleCharacterCasing="Normal"

                 Width="900"
                 Height="600"
                 WindowStartupLocation="CenterScreen"
                 mc:Ignorable="d">

...

</mah:MetroWindow>
```

![image](https://user-images.githubusercontent.com/658431/123074765-e5d1cb80-d417-11eb-876f-27f1c5ed3c29.png)

**Comment by timunie** [comment]
Hi @omeraziz 

I really think you should be more polite next time you ask a question. This is a free-time project which is given to you for free as in beer. More over you get support if you ask in a polite way. 

MahApps.Metro is a set of some Styles and additional controls in it's own design. If you don't like the design you have two options: 
1. Take MahApps and re-style the parts you don't like
2. Use another Framework which suits your needs better

I hope you got my point and if you like MahApps and are more kind next time, then you are really welcome 👍 . 

Happy coding 
Tim
